### DISCUSSION TITLE
After initial sensor installation there is Fault on Grid UI but so-status on host shows all green

### DISCUSSION THREAD

**Discussion by tone6740** [Security-Onion-Solutions/securityonion#13192] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

256

### Storage for /

293G

### Storage for /nsm

80TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Same issues as shown on this discussion https://github.com/Security-Onion-Solutions/securityonion/discussions/8417
After initial install sensor shows fault 100%CPU usage in GRID UI, but all green when so-status is run on the sensor 
![SecO_tshooting2](https://github.com/Security-Onion-Solutions/securityonion/assets/7986486/3a7c26a0-8d68-427d-9b71-e033ea8e5f43)
host

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by tone6740** [comment]
![SecO_tshooting3](https://github.com/Security-Onion-Solutions/securityonion/assets/7986486/8d876365-711e-4e3e-bc70-9cc0274fbe18)

**Comment by cm-ops** [comment]
Try running `sudo so-sensoroni-restart` on the sensor and see if that changes, if not check traffic to the manager on port 443, you could use `nc -vz $manager_IP 443` (replace $manager_IP with your actual manager's IP address) and see if it connects.

**Comment by tone6740** [comment]
![SecO_tshooting4](https://github.com/Security-Onion-Solutions/securityonion/assets/7986486/488a953f-21a8-4f42-b42c-6b580a4fc0e0)

nc fails but I'm able to curl telnet to 443 on the manager and I can also observe 443 traffic from the sensor on the management interface of the manager

I've reinstalled the image numerous times on this physical server but the same result, I've had similar problems with other physical servers(1 manager+search and 1 sensor) but after a few installs it just works(wish had a good reason for why the installation actually works)
This server on the other hand!! 
This server has 2 Intel(R) Xeon(R) Silver 4214 CPU @ 2.20GHz with 12 cores each(very much doubt CPU consumption is the issue here), any hardware compatibility issues known with this chip set or Dell PowerEdge R740XD2

**Reply by cm-ops** [reply]
Have you tried to tcpdump using the manager's IP and port 443? Also, do you see any issues in the` /opt/so/log/sensoroni/senoroni.log` on the sensor?
